### PR TITLE
[plugin.video.curiositystream] 1.0.0

### PR DESCRIPTION
**Pull request by GMaxera**
Description

This add-on enables browsing and playing content from your [Curiositystream](https://curiositystream.com) account

Zip package for testing the add-on: 
[plugin.video.curiositystream.zip](https://github.com/xbmc/repo-plugins/files/4432504/plugin.video.curiositystream.zip)

Checklist:
- [x] My code follows the add-on rules and piracy stance of this project.
- [x] I have read the CONTRIBUTING document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.curiositystream/main.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
+import functools
```
Comment (enen92) [inline plugin.video.curiositystream/main.py:1]: usually this file, as it is the entrypoint should be as small as possible. Just import the main codebase and run. The reason behind it is the fact that this is the only file that's not compiled to bitcode

File: plugin.video.curiositystream/main.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+import sys
+from contextlib import contextmanager
+from urlparse import parse_qsl
```
Comment (enen92) [inline plugin.video.curiositystream/main.py:4]: not python3 compatible, you might want to make it py2 and py3 compatible to ease the migration and future submission to the matrix branch

File: plugin.video.curiositystream/main.py (hunk 1 above)
Comment (GMaxera) [reply plugin.video.curiositystream/main.py]: I've reduced as much as possible 👍

File: plugin.video.curiositystream/main.py (hunk 2 above)
Comment (GMaxera) [reply plugin.video.curiositystream/main.py]: I've used try/expect but not sure if it's the best practice in that cases. Please, let me know

Comment (enen92) [reply plugin.video.curiositystream/main.py]:
```python
import sys

PY3 = sys.version_info.major

if PY3:
    your py3 import logic
else:
   your py2 import logic
```

Comment (GMaxera) [reply plugin.video.curiositystream/main.py]: Thanks :-) Done 👍

**Pull request merged**